commerce-cloud-data/your-shop-here-data/users.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<users xmlns="http://www.demandware.com/xml/impex/user/2007-09-05">
+    <user login="hnestmann@salesforce.com" enabled-flag="true">
```

Comment:
Do you really want to keep this here?

Reply:
good catch - only the demo user should be in there